defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 grafana_repo_release: stable
 grafana_service_name: grafana-server
```

Comment:
I can't understand why this variable is in `defaults`. It's `vars` obviously

Reply:
True. But not the scope of the PR :)